src/pages/journal/components/empty-slot/__mocks__/screen-orientation.mock.ts

```diff
@@ -0,0 +1,34 @@
+import { Observable } from 'rxjs/Observable';
+export class ScreenOrientationMock  {
```

Comment:
Did we try this? https://github.com/chrisgriffith/ionic-native-mocks

Reply:
the mocks folder and contents were duplicated by accident. have removed them